src/view/components/formikReactSpectrum3/textField.jsx

```diff
@@ -28,7 +28,9 @@ const TextField = ({ name, description, width, ...otherProps }) => {
       <ReactTextField
         {...otherProps}
         value={value}
-        onChange={setValue}
+        onChange={newValue => {
+          setValue(newValue);
+        }}
```

Comment:
Looks like you can revert this unless you want it for debugability or something.